.github/workflows/Release-helm-otelcollection.yml

```diff
@@ -0,0 +1,39 @@
+name: Main - Release Helm Charts - OtelCollection
+on: 
+  push:
+    branches: 
+      - main
+    paths:
+      # Only run this when chart file is updated.
+      - 'deployment/helm/otelcollection/Chart.yaml'
+      
+jobs:
+  Release-Artifacts:      
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Configure Git
+      run: |
+        git config user.name "$GITHUB_ACTOR"
+        git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+    - name: Setup Helm
+      uses: azure/setup-helm@v3.5 
+      with:
+        version: v3.9.2
+    
+    - name: Add Helm dependencies
+      run: |
+        helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+        helm repo add grafana https://grafana.github.io/helm-charts
+
+    - name: Run chart-releaser
+      uses: helm/chart-releaser-action@v1.1.0
+      env:
+        CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      with:
+        charts_dir: deployment/helm
```

Comment:
This is something we should discuss. The approach that has been taking with structure of the charts and template directories. I think we could take the cleaner approach per chart deployment, but this requires restructuring the base directory of each existing Helm chart due to where the Charts Releaser expects the directory structure to be for a group of charts or each separate chart.
It's an additional effort we can decide to do (will be cleaner)! Let's discuss :)

Reply:
Agreed with above; since we are adding more charts, the significance separating them out for independent release cycles is higher. Definitely a separate PR though.

Reply:
As agreed together, we need to setup a cleaner structure for all the Helm charts and create a single GH workflow chart releaser. This is now logger as a new issue to split up the work, it will be part of a new PR separate from the specifics of Observability work.